pkgs/top-level/all-packages.nix

```diff
@@ -18229,14 +18229,21 @@ in
 
   welle-io = libsForQt5.callPackage ../applications/radio/welle-io { };
 
-  wireshark = callPackage ../applications/networking/sniffers/wireshark {
+  inherit (callPackages ../applications/networking/sniffers/wireshark {
     inherit (darwin.apple_sdk.frameworks) ApplicationServices SystemConfiguration;
-  };
-  wireshark-qt = wireshark;
```

Comment:
Should an alias be added for `wireshark-qt`, in case folks are using it? As-is the attribute is dropped.

Reply:
sure, it was a typo